packages/nc-gui/components/project/AccessSettings.vue

```diff
@@ -107,8 +100,7 @@ const updateCollaborator = async (collab: any, roles: ProjectRoles) => {
     }
   } catch (e: any) {
     message.error(await extractSdkResponseErrorMsg(e))
-  } finally {
-    reloadCollabs()
+    loadCollaborators()
```

Comment:
why load on error? no need to reload after create / update?

Reply:
no need of reload when update since the UI is already updating when select is changed,  reloading API again causes whole table to rerender but on error we need to reset the select that's why we are reloading it in error